src/wmtk/invariants/CMakeLists.txt

```diff
@@ -54,6 +54,8 @@ set(SRC_FILES
 
     SmallerFunctionInvariant.hpp
     SmallerFunctionInvariant.cpp
+    uvEdgeInvariant.hpp
+    uvEdgeInvariant.cpp
```

Comment:
Is that supposed to be in here?

Reply:
I guess so, these two files exist in this folder.